L/libfyaml/bundled/patches/alloca.patch

```diff
@@ -0,0 +1,96 @@
+diff -ru a/include/libfyaml.h b/include/libfyaml.h
```

Comment:
It'd be great to upstream the patch

Reply:
I'll do that. (It's only needed for FreeBSD.)